acme/acme/challenges.py

```diff
@@ -410,7 +411,7 @@ class TLSALPN01Response(KeyAuthorizationChallengeResponse):
 
     """
 
-    ID_PE_ACME_IDENTIFIER_V1 = b"1.3.6.1.5.5.7.1.30.1"
+    ID_PE_ACME_IDENTIFIER_V1 = "1.3.6.1.5.5.7.1.30.1"
```

Comment:
hm...not sure we can change this before the major version update? I could see someone calling it and now hitting a type error. isn't all of `acme` in the public stable interface?

Reply:
yeah i think that's fair, i'll revert this